README.md

```diff
@@ -77,6 +79,35 @@ Here's an example of using it in nixos configuration.
 }
 ```
 
+### Example: A `devShell` Flake
+
+Assuming you have a valid `shell.nix`, running `nix develop` will create a shell with the default nightly Rust toolchain installed:
+
+```nix
+{
+  description = "Blah";
```

Comment:
Sorry, I was talking about the blah, and not the header.

Reply:
Haha what a silly my mistake

Reply:
Fixed

Reply:
Awesome!